docs/Reference/Validation-and-Serialization.md

```diff
@@ -165,7 +165,7 @@ of `'object'` and a `'properties'` object containing parameters) or a simpler
 variation in which the `type` and `properties` attributes are forgone and the
 parameters are listed at the top level (see the example below).
 
-> ℹ If you need to use the lastest version of Ajv (v8) you should read how to do
+> ℹ If you need to use the latest version of Ajv (v8) you should read how to do
 > it in the [`schemaController`](./Server.md#schema-controller) section. It is
 > explained the easier way to avoid to implement a custom validator.
```

Comment:
I have no idea what this sentence was trying to convey.

Reply:
remove?

Reply:
Removed in https://github.com/fastify/fastify/pull/3877/commits/482bf5782d4f04480c284d33bb96ac37fe74eb6f